client/src/pages/SigninContainer.js

```diff
@@ -7,197 +7,203 @@ import { Link } from "react-router-dom";
-								type="email"
-								color="secondary"
-								error={
-									state.emailError || state.emailValidationError ? true : false
-								}
-								helperText={
-									state.emailError || state.emailValidationError
-										? state.emailError
-											? "This email does not exist in our records."
-											: "Please enter a valid email address."
-										: ""
-								}
-								onChange={handleInputChange}
-							/>
-						</Grid>
-						<Grid item xs={12}>
-							<TextField
-								variant="outlined"
-								required
-								fullWidth
-								label="Password"
-								value={state.pwdSignin}
-								id="pwdSignin"
-								name="pwdSignin"
-								type="password"
-								color="secondary"
-								error={
-									state.pwdError || state.pwdValidationError ? true : false
-								}
-								helperText={
-									state.pwdError || state.pwdValidationError
-										? state.pwdError
-											? "You have entered an incorrect password."
-											: "Please enter a password."
-										: ""
-								}
-								onChange={handleInputChange}
-							/>
-						</Grid>
-					</Grid>
-				</form>
-				<Box textAlign="center">
-					<Button
-						type="submit"
-						variant="contained"
-						color="primary"
-						size="large"
-						classes={{ root: classes.modalSubmit }}
-						onClick={handleSubmit}
-					>
-						Sign In
-					</Button>
-				</Box>
-			</div>
-			<hr />
-			<div className="modal-footer">
-				<Typography
-					component="p"
-					variant="subtitle1"
-					align="center"
-					className={classes.modalFooter}
-				>
-					Don't have an account?{" "}
-					<Link to="/signup" className={classes.link}>
-						Sign Up
-					</Link>
-				</Typography>
-			</div>
-		</Container>
-	);
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const { emailSignin, pwdSignin } = state;
+    const userInfo = {
+      email: emailSignin.trim().toLowerCase(),
+      password: pwdSignin.trim().toLowerCase(),
+    };
+    if (checkEmail() && pwdSignin) {
+      fetch("/api/users/login", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(userInfo),
+      })
+        .then((res) => res.json())
+        .then((results) => {
+          if (results.status === "success") {
+            redirect(); // redirect to dashboard
+            // User object
+            console.log(results);
+          } else {
+            // Errors object
+            console.log(results);
+          }
+          // else if ("email" in results) {
+          // 	dispatch({ type: "error", error: "emailError" });
+          // } else if ("password" in results) {
+          // 	dispatch({ type: "error", error: "pwdError" });
+          // } else if ("errors" in results) {
+          // 	if ("email" in results.errors) {
+          // 		dispatch({ type: "error", error: "emailValidationError" });
+          // 	}
+          // }
```

Comment:
remove comments. we can always come back and grab them from a previous commit

Reply:
This is not my code. Is it okay if I delete it?

Reply:
You can delete it.